project14-Whack-a-Penguin/WhackSlot.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  WhackSlot.swift
+//  project14-Whack-a-Penguin
+//
+//  Created by Felipe Gil on 2021-08-05.
+//
+
+import UIKit
+import SpriteKit
+
+class WhackSlot: SKNode {
+    var charNode: SKSpriteNode!
+    var isVisible = false
+    var isHit = false
+    
+    
+    func configure(at position: CGPoint) {
+        self.position = position
+        
+        let sprite = SKSpriteNode(imageNamed: "whackHole")
+        addChild(sprite)
+        
+        let cropNode = SKCropNode()
+        cropNode.position = CGPoint(x: 0, y: 15)
+        cropNode.zPosition = 1
+        cropNode.maskNode = SKSpriteNode(imageNamed: "whackMask")
+        
+        charNode = SKSpriteNode(imageNamed: "penguinGood")
+        charNode.position = CGPoint(x: 0, y: -90)
+        cropNode.addChild(charNode)
+        
+        addChild(cropNode)
+    }
+    
+    func show(hideTime: Double) {
+        if isVisible { return }
+        charNode.xScale = 1
+        charNode.yScale = 1
+        
+        charNode.run(SKAction.moveBy(x: 0, y: 80, duration: 0.05))
+        isVisible = true
+        isHit = false
+        
+        if Int.random(in: 0...2) == 0 {
+            charNode.texture = SKTexture(imageNamed: "penguinGood")
```

Comment:
I don't think it's worth changing here, especially because you did that in another exercise, but we try to avoid using strings directly, especially when we need to do it multiple times (you have "penguinGood" here and in line 28).

Reply:
I'll change it! 
It's important to practice a good way of programming. 
Fixing! 
Thank you!